lib/gamnit/flat.nit

```diff
@@ -1077,7 +1077,7 @@ end
 #
 # array.add "E"
 # assert array.to_s == "[A,B,c,D,E]"
-# assert array.capacity == 5
+# assert array.capacity == 6
```

Comment:
Because there may be holes between items, this capacity is quite significant, it reflects the minimum allocated space on the GPU. We free up capacity only by calling `defragment` or when completely clearing the `GroupedArray`. The error in the unit test was caused by a change in this behavior.

I prefer to test this value to detect useless grow and as any change may cause much harder to debug OpenGL errors.